mitodl/bootcamp-ecommerce
# feat: add early bird deadline

#### What are the relevant tickets?
- https://github.com/mitodl/hq/issues/2941
- https://github.com/mitodl/hq/issues/2943

#### What's this PR do?
- Removes the deadline from the payment section in a bootcamp application
- Adds early bird deadline in a bootcamp application in the video interview section of the application


#### How should this be manually tested?

- Enroll in any bootcamp
- Use `set_application_state` command to set the user's application state if necessary
- Check that there is no deadline shown on the payment section of the application
- Set a date for the early bird deadline in bootcamp run
- Check that you see `Early Bird Deadline` in the `Video Interview section` of the Bootcamp application
- Now set the application state to `Video Interview completed`, You should not see an early bird deadline now



#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)

**When early bird deadline is set**
<img width="1300" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/d56f1e64-41ef-4cad-a6ef-7554efce441b">

---

**When early bird deadline is set**
<img width="1381" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/078a8cd8-03d0-4bd3-8945-ceae466907e9">

---

**When the status is completed** (No deadline will be shown even set)
<img width="1114" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/6cfbe380-bae0-41cb-81d1-efad8d6cdd64">
